R/util_dfc.R

```diff
@@ -0,0 +1,238 @@
+##########################################################################
+###
+###     util_dfc.R
+###     Dan Greene, 12/21/17
+###
+###     The goal of the util_dfc ("data frame compare") module is to make it easy to see
+###     where two data frames differ for debugging and further analysis.
+###
+###     Depends on util.R
+###
+
+util_dfc.compare_vecs <- function(vec1, vec2) {
```

Comment:
I'd make the module prefix here a little simpler, maybe just `dfc` (our scale computation modules are just `sc.function` and our graphics functions are just `ug.function`, data summarise functions are `ds.function` etc.)

Reply:
love it